### ISSUE TITLE
Is there mouse acceleration enabled?

### ISSUE THREAD

**Issue by jberq** [yquake2/yquake2#1084]
I looked through the documentation and couldn't find anything relating to mouse accel. This mouse movement doesn't quite feel raw to me— is this a placebo effect, or is there some kind of acceleration applied to mouse movement?

**Comment by Yamagi** [comment]
Indeed, we are using relative mouse input. YQ2 reads how many pixels the mouse has moved and calculates the relative ingame movement from that. YQ2 itself hasn't support for mouse acceleration in the modern sense. 

There is `exponential_speedup` which implements a kind of poor mans acceleration, however it's disabled by default and seems to be broken. And there the good, old `m_filter` which tries to filter out inaccuracies caused by ball mice. Both are disabled by default.

If you are experiencing mouse acceleration it's likely caused by something outside of YQ2, maybe mouse software or another bug in SDL.
